### PR TITLE
Add GH Actions workflow to build PR

### PR DESCRIPTION
**Pull request by evidolob**
It is simple Action to build each PR across all OS.

### CODE REVIEW

**Comment by joshuawilson** [comment]
What is Travis used for it you switch to github actions?

File: .github/workflows/CI.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, windows-latest, ubuntu-18.04]
```
Comment (joshuawilson) [inline .github/workflows/CI.yaml:21]: Why is the version set for ubuntu, instead of `ubuntu-latest`?

**Comment by evidolob** [comment]
Now it is using for publishing npm package, as it is dome from @gorkem account.
I think for publishing we need to have separate workflow

File: .prettierrc.json [hunk 2]
```diff
@@ -1,6 +1,8 @@
 {
+  "$schema": "http://json.schemastore.org/prettierrc",
   "arrowParens": "always",
   "printWidth": 130,
   "singleQuote": true,
-  "trailingComma": "es5"
+  "trailingComma": "es5",
+  "endOfLine": "lf"
```
Comment (joshuawilson) [inline .prettierrc.json:7]: `"endOfLine": "auto"` works for me across the different OS'es

**Pull request merged**